### ISSUE TITLE
Feature: ios:accessibilityPerformEscape()

### ISSUE THREAD

**Issue by m-abs** [m-abs/nativescript-accessibility-ext-private#8]
**accessibilityPerformEscape()**

Dismisses a modal view and returns the success or failure of the action.

*User action:* A two-finger Z-shaped gesture that dismisses a modal dialog, or goes back one level in a navigation hierarchy. Returns true, if we're handling the escape. Dev note: Unlike accessibilityPerformMagicTap() this cannot be added to the AppDelegate in NativeScript.

**Note:** PR https://github.com/NativeScript/NativeScript/pull/4382 for adding to `ui/frame`

**Comment by m-abs** [comment]
We've moved to https://github.com/Notalib/nativescript-accessibility-ext
